lib/expressroutes.js

```diff
@@ -104,8 +114,8 @@ function makeValidator(validator, consumes) {
 function makeExpressRoute(router, mountpath, route) {
     var path, args, before, validators;
 
-    path = route.path.replace(/{([^}]+)}/g, ':$1');
```

Comment:
Why did you make a method of this ? It seems that there was a change in the Regexp but hard to see.